setup.cfg

```diff
@@ -64,3 +64,5 @@ ignore_missing_imports = True
 
 [mypy-urllib3.util.retry]
 ignore_missing_imports = True
+[metadata]
```

Comment:
We actually don't need this part; it seems that files matching `LICEN[CS]E*` are automatically included: https://wheel.readthedocs.io/en/latest/user_guide.html#including-license-files-in-the-generated-wheel-file